test/test_select_multiple.py

```diff
@@ -26,16 +27,16 @@ class TestSelectMultiplePrint(unittest.TestCase):
     def test_list_newlines(self, mock_print):
```

Comment:
Maybe add a second test (`test_list_newlines`, `test_list_newlines_hide_confirm`) here that tests for the default behaviour, with `hide_confirm=True`